catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -90,6 +90,7 @@
                 <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
                 <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
                 <bean class="ddf.catalog.data.impl.types.DateTimeAttributes"/>
+                <bean class="ddf.catalog.transformer.common.tika.Mp4MetacardType"/>
```

Comment:
This PR relies on the changes in PR #1601 so this change is appropriate given that constraint.